src/Pillar-Project.package/PRProject.class/instance/tableOfContents.st

```diff
@@ -3,7 +3,7 @@ tableOfContents
 	| tocFilePath tocFileName inputDoc |
 	tocFilePath := self configuration propertyAt: 'tocFile' ifAbsent: [ ^ PRToc new ].
 	tocFileName := self class pathToString: tocFilePath.
-	inputDoc := PRPillarParser parse: (self baseDirectory / tocFileName) contents.
+	inputDoc := PRPillarParser new optimize parse: (self baseDirectory / tocFileName) contents.
```

Comment:
Here, again, calling PRPillarParserMain should fix the problem, and again, this was like this before Juraj's arrival, not his fault, but we need to fix it to avoid the extra calls to optimize.

Reply:
changed